File: Android applications/p0261intentfilter/java/com/korobeynikov/p0261intentfilter/ActivityDate.java
package com.korobeynikov.p0261intentfilter;

import java.sql.Date;
import java.text.SimpleDateFormat;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

public class ActivityDate extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.date);
        SimpleDateFormat sdf=new SimpleDateFormat("dd.MM.yyyy");
        String date=sdf.format(new Date(System.currentTimeMillis()));
        TextView tvDate=findViewById(R.id.tvDate);
        tvDate.setText(date);
    }
}

File: KotlinLessons/AndroidLessons/p0261intentfilter/java/com/korobeynikov/p0261intentfilter/ActivityDate.kt
package com.korobeynikov.p0261intentfilter

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.date.*
import java.text.SimpleDateFormat
import java.util.*

class ActivityDate : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.date)

        val sdf=SimpleDateFormat("dd.MM.yyyy")
        val date=sdf.format(Date(System.currentTimeMillis()))
        tvDate.text=date
    }
}